packages/components/src/tabs/stories/index.story.tsx

```diff
@@ -367,133 +366,3 @@ const CloseButtonTemplate: StoryFn< typeof Tabs > = ( props ) => {
 	);
 };
 export const InsertCustomElements = CloseButtonTemplate.bind( {} );
-
-const ControlledModeTemplate: StoryFn< typeof Tabs > = ( props ) => {
```

Comment:
Controlled mode is explained better on the Best Practices page. Behavior should/can be covered in the unit tests.